locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -134,17 +134,16 @@ EXPOSE 8080
 ```
 
 Last but not least, define the command to run your app using `CMD` which defines
-your runtime. Here we will use the basic `npm start` which will run
-`node server.js` to start your server:
+your runtime. Here we will use `node server.js` to start your server:
 
 ```docker
-CMD [ "npm", "start" ]
+CMD [ "node", "server.js" ]
```

Comment:
Nitpicking perhaps but maybe this should be `script.js` or `index.js` since not every node app is a server?

Reply:
Hi Laurent, thanks for the feedback. It should stay as-is because the guide, as it is written today, is a step-by-step guide that explicitly sets up a server and instructs the reader to create `server.js`. This PR is simply updating the pattern in the code without changing the scope of guide itself. Here's a [link to the guide for reference](https://nodejs.org/en/docs/guides/nodejs-docker-webapp/).